server/out/IonText.interp

```diff
@@ -0,0 +1,97 @@
+token literal names:
```

Comment:
this is under `server/out` which I think we do not need. `server/out` is the build output and we should not add it to the repository